packages/node/src/integrations/undici/index.ts

```diff
@@ -36,6 +37,13 @@ export enum ChannelName {
   RequestError = 'undici:request:error',
 }
 
+const _undiciIntegration = ((options?: UndiciOptions) => {
+  // eslint-disable-next-line deprecation/deprecation
+  return new Undici(options) as unknown as IntegrationFnResult;
+}) satisfies IntegrationFn;
+
+export const undiciIntegration = defineIntegration(_undiciIntegration);
```

Comment:
we should probably rename this right now tbh. Nobody recognizes Undici 😢

Reply:
ahh right! Let's go with `nativeNodeFetchIntegration()` as Luca proposed some time ago 👍

Reply:
Renamed it!